lib/components/number_Button.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+class NumberButton extends StatelessWidget {
+  final String text;
```

Comment:
At this point, you already know it is a number. So take int as parameter, not String